utils/cmake/dreamcast.cmake

```diff
@@ -1,5 +1,23 @@
+# Auxiliary CMake Utility Functions
+#   Copyright (C) 2023 Colton Pawielski
+#   Copyright (C) 2024 Falco Girgis
+#
+# This file implements utilities for the following additional functionality
+# which exists in the KOS Make build system:
+#   1) linking to existing binaries
+#   2) adding a romdisk
+#
+# NOTE: When using the KOS CMake toolchain file, you do not need to include
+#       this file directly!
+
 cmake_minimum_required(VERSION 3.23)
 
+#### Set Configuration Variables From Environment ####
+if(NOT DEFINED ENV{KOS_BASE}
+   OR NOT DEFINED ENV{KOS_CC_BASE})
+    message(FATAL_ERROR "KallistiOS environment variables not found")
```

Comment:
Here we test the env variable for KOS_CC_BASE, but below at line 82 its using the non-env variable ( which can be not set if we don't pass by the toolchain.cmake.  Is the case for example in my project as I was using a custom toolchain.cmake)